lib/mobility/backends/active_record/container.rb

```diff
@@ -107,32 +108,22 @@ def initialize_dup(source)
       private
 
       def model_translations(locale)
-        model[column_name][locale] ||= {}
-      end
+        translations = model[column_name]
 
-      def set_attribute_translation(locale, value)
-        translations = model[column_name] || {}
-        translations[locale.to_s] ||= {}
-        translations[locale.to_s][attribute] = value
-        model[column_name] = translations
-      end
+        return unless translations
 
-      class Coder
-        def self.dump(obj)
-          if obj.is_a? ::Hash
-            obj.inject({}) do |translations, (locale, value)|
-              value.each do |k, v|
-                (translations[locale] ||= {})[k] = v if v.present?
-              end
-              translations
-            end
-          else
-            raise ArgumentError, "Attribute is supposed to be a Hash, but was a #{obj.class}. -- #{obj.inspect}"
-          end
-        end
+        translations[locale.to_s]
```

Comment:
We don't need to be defensive here: the default `{}` should be on the column, so this can simply be:

```ruby
model[column_name][locale.to_s]
```